config/config.go

```diff
@@ -367,6 +388,8 @@ func ParseTunnelProto(proto string) TunnelProto {
 		return TCP
 	case "tls":
```

Comment:
See above for TLS details. In this commit TLS over TCP Handlers doesn't work anymore. My plan was to address this in another PR in relation to https://github.com/superfly/wormhole/issues/11